script/node-disabled-tests.json

```diff
@@ -87,6 +87,7 @@
   "parallel/test-tls-cnnic-whitelist",
   "parallel/test-tls-disable-renegotiation",
   "parallel/test-tls-empty-sni-context",
+  "parallel/test-tls-error-stack",
```

Comment:
Disabling this isn't a blocker, but is there more context about why this is disabled? e.g. is it a new regression, or is this a new test that was just added upstream, or...?